schema/transportation/segment.yaml

```diff
@@ -51,6 +57,71 @@ properties:
         default: []
 "$defs":
   propertyDefinitions:
+    destinationLabelType:
+      description: >-
+        The type of object of the destination label.
+      type: string
+      enum: 
+      - place
+      - street
+      - country 
+      - route
+    destinations:
+      description: >-
+        Information about what destinations can be reached if following this segment. 
+        Includes references to the segments from which it is needed to transition 
+        onto this segment to reach the destinations.
+      type: array
+      items:
+        type: object
+        unevaluatedProperties: false
+        required: [labels, from, to_connector_id, final_heading]
+        properties:
+          labels:
+            description: >-
+              Describes the destination labels on the directions sign.
+            type: array
+            items:
+              type: object
+              unevaluatedProperties: false
+              required: [value, type]
+              properties:
+                value:
+                  description: >-
+                    Names the object that is reached
+                  type: string
+                  pattern: ^(\S.*)?\S$
+                type: { "$ref": "#/$defs/propertyDefinitions/destinationLabelType" }
+            minItems: 1
+            uniqueItems: true
+          symbols:
+            type: array
+            items:
+              description: >-
+                A symbol or icon present on the sign.
+              type: string
+              pattern: ^(\S.*)?\S$
```

Comment:
My first thought was, how can a string be a symbol?

My second thought was, shouldn't this be enumerated?

If I'm looking at the right TagInfo, there don't seem to be a an enormous number of alternatives: https://taginfo.openstreetmap.org/keys/destination%3Asymbol#values

Reply:
replaced with enum